pkg/generate/app/environmentfile.go

```diff
@@ -0,0 +1,38 @@
+package app
+
+import (
+	"bufio"
+	"os"
+)
+
+// IsPossibleEnvironmentFile returns true if the argument is a file. I am not
+// entirely sure, why we are using this function, as this is just a wrapper,
+// we are not doing any additional validations here. This is something that I
+// am borrowing from templates file. Need feedback on this.
+func IsPossibleEnvironmentFile(value string) bool {
+	var fileexists bool = true
+	_, err := os.Stat(value)
+	if err != nil {
+		fileexists = false
+	}
+	return isFile(value) && fileexists
+}
+
+// getEnvironmentVars reads a whole file into memory and returns string
+// array. This could be a problem if we are reading a very large file. I am
+// not putting any validation for file, if it contains a key=value pair as
+// that should be handled in environment variables that we are passing.
+func GetEnvironmentVars(path string) []string {
```

Comment:
it would be worth doing validation here because if you wait until you're checking the environment variables, you can't give the user a useful error message that says "your env file contains invalid content", all you can do is say "blahblah is not a valid key=value pair" but the user doesn't immediately know where it came from.